config.go

```diff
@@ -27,10 +27,14 @@ import (
 const configVersion = "unstable"
 
 type config struct {
-	Version   string
-	Generator string
```

Comment:
Would it be possible to keep compatibility with the previous syntax?

Reply:
It may be. Let me check.

Reply:
The latest revision should support both Generator and Generators. The former is converted to the latter internally.